src/java/com/wolfssl/provider/jsse/WolfSSLSocket.java

```diff
@@ -2383,7 +2383,12 @@ public synchronized int read() throws IOException {
             byte[] data = new byte[1];
 
             try {
-                this.read(data, 0, 1);
+                int ret = this.read(data, 0, 1);
```

Comment:
Can you declare `int ret;` at the top of the method, above `byte[] data = new byte[1]`?  That will make us consistent in putting variable declarations at the top of the method in most cases.